src/runtimes/node/bundler.js

```diff
@@ -8,9 +8,17 @@ const semver = require('semver')
 
 const { getBundlerTarget } = require('./bundler_target')
 const { externalNativeModulesPlugin } = require('./native_modules/plugin')
+const { processSourcemap } = require('./sourcemap')
 
 const pUnlink = promisify(fs.unlink)
 
+const supportsAsyncAPI = semver.satisfies(process.version, '>=9.x')
```

Comment:
No functionality has changed here. Extracted from `bundleJsFile` to conform to the linter rules.